### PR TITLE
updated ci to the latest org

### PR DESCRIPTION
**Pull request by cindyuc**

### CODE REVIEW

File: ci/main.go [hunk 1]
```diff
@@ -71,10 +71,8 @@ func build(ctx context.Context) error {
...
 	// mount a host directory in the container at the '/workspaces/parigot' path
-	img := client.Container().
-		Build(dockerDir).
-		WithDirectory(
-			"/workspaces/parigot",
+	img := dockerDir.DockerBuild().
+		WithDirectory("/workspaces/parigot",
 			client.Host().Directory("."),
 			dagger.ContainerWithDirectoryOpts{Exclude: []string{".devcontainer/", "ci/", "build/", "g/"}},
 		).
```
Comment (iansmith) [inline ci/main.go:78]: I'm not totally sure what the point is of the `WithDirectoryOps` but there is a lot of files in `g/` that are dependencies for other parts of the system.  If that's not what you are doing this for, you can ignore this comment.

**Review by iansmith — APPROVED** [review]
All seems fine to me.  Just take a look at the comment and ignore if you think it's not important.

Does this change in the directories mean you can use it from _inside_ the dev container?

File: ci/main.go (hunk 1 above)
Comment (cindyuc) [reply ci/main.go:78]: - `WithDirectory` will mount the source code directory on the host at the `/workspaces/parigot/` mount point in the container
- `dagger.ContainerWithDirectoryOpts{Exclude: []string{".devcontainer/", "ci/", "build/", "g/"}}` aims to exclude these files from the mount. 

Files in `g/` will be generated at build time and I don't think they need to be mounted.

**Comment by cindyuc** [comment]
> All seems fine to me. Just take a look at the comment and ignore if you think it's not important.
> 
> Does this change in the directories mean you can use it from _inside_ the dev container?

Unfortunately, I still need to try some ways to get it to run inside the dev container. These changes are just to make it work well in the current version

**Pull request merged**